javalib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -163,6 +240,25 @@ class Throwable(s: String, private var e: Throwable)
     duplicates
   }
 
+  def setStackTrace(stackTrace: Array[StackTraceElement]): Unit = {
+
+    if (stackTrace eq null) {
+      throw new java.lang.NullPointerException()
+    }
```

Comment:
This is redundant. Accessing `stackTrace.length` will itself throw an NPE if `stackTrace` is `null`.

Reply:
Removed, thank you.